web_frontend/cloudscheduler/csv2/templates/csv2/status.html

```diff
@@ -448,17 +492,7 @@
                                 {% if cloud.display >= 9 %}
                                     <td class="bold-style" onclick="toggle_id('expand-{{cloud.group_name}}-{{cloud.cloud_name}}')"><div class="float-left">Totals</div>
                                 {% elif cloud.enabled > 0 %}
-                                    {% if cloud.app_credentials_expiry and cloud.current_time %}
```

Comment:
Seems like you may have been working with an older version of this file as your merge request removes a lot of the stuff added in 2.9.0, the application credential stuff and the volume stuff should be in the final version. You may want to merge 2.9.0 into your branch first then make a new pull request with your changes.

Reply:
stable-2.9 has been merged and the missing volume stuff added manually